tests/unit/command/ls/test_ls_colors.py

```diff
@@ -0,0 +1,128 @@
+from dvc.command.ls.ls_colors import LsColors
+from dvc.path_info import PathInfo
+
+
+def test_ls_colors_out_file():
+    ls_colors = LsColors()
+    ls_colors.load(LsColors.default)
+    output = ls_colors.format(
+        {
+            "path_info": PathInfo("file"),
+            "isexec": False,
+            "isdir": False,
+            "isout": True,
+        }
+    )
```

Comment:
May use a template function to make tests smaller/more glancable:
```python
assert _colored("file", "d") == "file"
```
The second argument being a string containing any subset of `"edo"`.